extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,39 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+let blbSlot = {
+	config: {
+		targeting: {}
+	},
+	getSlotName: () => 'BOTTOM_LEADERBOARD',
+	getId: () => 'BOTTOM_LEADERBOARD'
+};
+
+Context.extend(config);
+TemplateService.register(BigFancyAdBelow);
+SlotService.add(blbSlot);
+
+function loadCustomAd(callback) {
```

Comment:
can we rename `callback` to `fallback` or something that tells it's a backup plan?